docs/release-notes/2.31.0.md

```diff
@@ -0,0 +1,157 @@
+# Percona Monitoring and Management 2.31.0
+
+| **Release date:** | Sep 26, 2022                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of PMM. This ensures that you have access to the latest PMM features and that your environment runs on the latest version of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+
+### Enhaced query building, navigation and Alerting
+
+We have upgraded Grafana to version 9.1 in PMM 2.31.0. With this upgrade, you can get a simplified user interface, enhanced visualizations, and a default unified Alerting experience. For more information, see [What’s new ](https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v9-1/) in Grafana.
+ 
+The following features have been introduced with PMM 2.31.0:
+ 
+- **Expandable navbar (main menu)**
```

Comment:
can we update here as main menu instead navbar?

Reply:
Done